app/blog/20161005-frontend-build-instructions

```diff
@@ -0,0 +1,81 @@
+---
+title: Welcome to the n.io Front-End Team!
+author:  Trisha Allin
+posted: 20161005
+summary:
+  Everything you need to get your front-end build up and running.
+_options:
+  layout: app/layouts/blog.static.hbs
+---
+
+
+## Install your favorite shell script and text editor!
+Feel free to use whichever terminal shell and text editor you work with best.
+
+Recommendations:
+1. Shell script: zsh/oh-my-zsh
+This shell allows you to theme, has helpful plug-ins and shows you which branch you are on.
+https://github.com/robbyrussell/oh-my-zsh
+2. Text editor: Sublime Text, Atom
+Patrick will probably supply you with sublime… but here’s the link just in case
+https://www.sublimetext.com/
+
+
+
+## Let’s Install Homebrew!
+http://brew.sh/index.html
+
+Paste this into your terminal window at the root level:
+
+`/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+
+
+## Let’s Install Node.js!
+https://nodejs.org/en/
+
+Now that you have Homebrew all you have to do is type in the terminal:
+
+`brew install node`
+
+## Let’s Get Synced with GitHub!
+###Generate your ssh key
+https://help.github.com/articles/generating-an-ssh-key/
+1. See if you already have an ssh key by typing in terminal: ls -al ~/.ssh
+Generate a new key by typing: 
+`ssh-keygen -t rsa -b 4096 -C "your_email@example.com". `
+2. Press Enter to accept default location
+3. Choose and enter a passphrase
+###Add your ssh key to the agent 
+1. Make sure ssh agent is enabled by typing: 
+	`eval "$(ssh-agent -s)"`
+
+2. Add ssh key to the agent by typing: 
+	`ssh-add ~/.ssh/id_rsa`
+
+### Copy Key to GitHub Account
+1. Copy key to clipboard by typing: 
+	`pbcopy < ~/.ssh/id_rsa.pub`
+2. Paste key in the settings menu in GitHub
+
+
+## Let’s Install Brunch!
+http://brunch.io/
+1. Type in the console: 
+`brunch install -g`
+*Potential pitfalls: If you have issues installing brunch, try uninstalling node and reinstalling it
+2. Now install your dependencies with: 
+`npm install`
```

Comment:
You run 'npm install' whenever there is a package.json file. Pretty sure this is for the project level, not the global install.... My understanding is that when you navigate to your brunch project branch, that is where you type 'npm install' to install the dependencies for that branch/project.